addons/product/models/product_pricelist.py

```diff
@@ -299,7 +299,7 @@ def _get_partner_pricelist(self, partner_id, company_id=None):
         Property = self.env['ir.property'].with_context(force_company=company_id or self.env.user.company_id.id)
 
         p = Partner.browse(partner_id)
-        pl = Property.get('property_product_pricelist', Partner._name, '%s,%s' % (Partner._name, p.id))
+        pl = Property.get('property_product_pricelist', Partner._name, False, '%s,%s' % (Partner._name, p.id))
```

Comment:
Pricelists shall remain individual as per company, not accounting company, hence: `False`.
Case:
Subsidiary 1: Onlineshop with distribution center
Subsidiary 2: Retail in a rich part of the country
Subsidiary 3: Retail in a poor part of the country